README.md

```diff
@@ -161,7 +162,7 @@ Add the following as another function in the App class:
 
 
 1. Run the server project in Xcode
-    1. In the top left corner of Xcode you should a small terminal icon with the text "FoodServer-Package" next to it. Click this icon and then click "FoodServer" from the dropdown menu.
+    1. In the top left corner of Xcode you should a small toolbox icon with the text "FoodServer-Package" next to it. Click this icon and then click "FoodServer" from the dropdown menu.
```

Comment:
Missing the word 'see' - you should see

Reply:
For the CompletedFoodTracker - please can you update the version of Kitura to the latest version (as it's still displaying the old splash page).